TestFlightAPI/TestFlightAPI/TestFlightAPI.cs

```diff
@@ -224,6 +224,18 @@ internal static string UnitStringForMTBFUnitShort(MTBFUnits units)
                     return "-";
             }
         }
+        
+        // Simply converts the failure rate to the reliability rate at time t.
+        public static float FailureRateToReliability(float failureRate, float t)
+        {
+            float reliability = Mathf.Exp(-failureRate * t);
+            return reliability;
+        }
+        public static double FailureRateToReliability(double failureRate, float t)
```

Comment:
Why do we need both forms?

Reply:
Convenience really, we could just cast everything to double (or float) though.